cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -12,6 +12,12 @@
   expect(num_matches_since_savepoint(message)).to be >= 1
 end
 
+And(/Alice's API key does not appear in the log/) do
+  expect(
+    num_matches_since_savepoint(api_key_for_role_id("cucumber:user:alice"))
+  ).to be 0
```

Comment:
We are leaning towards preferring parens around method calls, right? So this should be `be(0)`.